javascript/webapp/src/pages/HomePage.constants.ts

```diff
@@ -15,7 +15,7 @@ export const topLevelQuestionCards: CardDetail[] = [
 		questionType: QuestionType.Discovery,
 		heroTitle: pages.discover.title,
 		title: 'Do dataset features cause each other?',
-		href: pages.discover.route,
+		key: 'discover',
```

Comment:
These should (eventually) match the Resource profile key for each. I also wonder if we might want to use a convention to avoid potential collision, such as 'showwhy-exposure', 'showwhy-discover', 'showwhy-events' for these profiles. Obviously a super clear option would be to use the schema URL, but that somewhat defeats the purpose of the simpler profile name.

Reply:
Yep, that's a good point, this is all super in-flux still